services/header/rocketjoe/services/python_sandbox/data_set.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include <nlohmann/json.hpp>
 
+PYBIND11_DECLARE_HOLDER_TYPE(T, boost::intrusive_ptr<T>);
```

Comment:
Move to file forward.hpp .

Reply:
```C++
template <typename T>
using ref_counter = boost::intrusive_ref_counter<T, boost::thread_safe_counter>;
using  boost::intrusive_ptr; 
```

Reply:
So, most likely, it won't compile, because the compiler reports an error with the constructor or templates.